### PR TITLE
The Audio Update Pt. 1: Third Coming

### PR DESCRIPTION
**Pull request by nauticall**
## About The Pull Request
**Before I get assassinated by maintainers, let it be known that Github literally prohibits me from reopening #283 because I force-pushed to it while it was closed.** So I had to make a new PR for this if this is to be reopened and all. Funky!

See https://gitlab.com/cmdevs/colonial-warfare/-/merge_requests/1688 for the original thing as well as sound samples.

The culmination of like 20 combined hours of combing the A:CM sound library and cherrypicking sounds to implement into CM. The files were all numbered (not labeled) so it was an absolute pain to sort through everything and see which ones fit where.

Some of the sounds may be controversial, so let me know if anything seems better off as it was old.\

Per developer and community request this version does not come with a bloated changelog, extra voice lines, that beefy hijack thing, or that funky acid sound.

**7/3/2022:** Reopened! The bugs addressed in #283 have been fixed and addressed, and sounds that were not supposed to be there were removed.

## Why It's Good For The Game

This PR aims to bring a fresh coat of paint to the sounds you've spent years listening to over and over again. More stuff has more bass, no longer sounds like a watermelon being slapped with a stick, and overall gives better sound quality, hopefully leading to more immersion.

Hopefully.

I intend for this PR to be testmerged so that people can judge what sounds they like, what sounds they don't like, and what they'd like added.

In the future I'd like to expand on even more sounds, but this PR is a bit bloated as-is already.

## Changelog

:cl:
add: Added more than 150 new sounds into the game; most were ported from A:CM and stock SS13.
add: New firing sounds for the M37, Mk221/Mk210, M4A3, and 88 Mod 4.
add: New sounds for shotgun shell loading, glass breaking, explosions, and explosions greater than 300 power, e.g. OBs and max caps.
add: Added various new interaction noises, particularly when damaging barricades, prying doors, and picking up/dropping items.
add: Several tools now have pickup and drop sounds. In addition to the toolbelt tools, these also include binoculars and fire extinguishers.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/defibrillator.dm [hunk 1]
```diff
@@ -77,11 +78,11 @@
 	ready = !ready
 	user.visible_message(SPAN_NOTICE("[user] turns [src] [ready? "on and takes the paddles out" : "off and puts the paddles back in"]."),
 	SPAN_NOTICE("You turn [src] [ready? "on and take the paddles out" : "off and put the paddles back in"]."))
+	playsound(get_turf(src), "sparks", 25, 1, 0)
 	if(ready)
-		w_class = SIZE_LARGE
```
Comment (stanalbatross) [inline code/game/objects/items/devices/defibrillator.dm:81]: WHYYYYYYYYYYYYYYYYYYYYY

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Comment by stanalbatross** [comment]
some of the sounds also seem to be delayed on playing (particularly the "opening pouches" and "opening defibs" sound)

this is a bit inconsistent

**Comment by stanalbatross** [comment]
cade hitting sounds were reported as being hard to notice and a bit quiet

**Comment by nauticall** [comment]
> some of the sounds also seem to be delayed on playing (particularly the "opening pouches" and "opening defibs" sound)

They're totally fine on my end, I don't know what's going on with that >.>

**Comment by AGasCanister** [comment]
experienced some sounds and I can say it's beautiful, I really liked them
though I must say placing items or taking items out of tables is a bit weird

**Comment by Wintermote** [comment]
Some sounds were very quiet, and other sounds very loud - maybe needs loudness normalization; it might also be a problem with how the sounds are played in-game rather than anything with the audio directly.

Biggest change I noticed and liked were the sounds for moving tools around and drawing/wielding guns

**Comment by stanalbatross** [comment]
the sound of xenos breaking chairs is quite weird

**Comment by stanalbatross** [comment]
rob warraker says : I hate that every single action a marine does makes a sound

**Comment by stanalbatross** [comment]
glass breaking is too loud

**Comment by stanalbatross** [comment]
new explosions sound good; new shotgun sounds described as less impactful

**Comment by stanalbatross** [comment]
wallace haar says : the overabundance of sound makes it less impactful

**Comment by stanalbatross** [comment]
lumdor says : the new slash sounds are less meaty

**Comment by stanalbatross** [comment]
the fact that moving items around in storage always makes a sound does get annoying

**Comment by stanalbatross** [comment]
xeno door forcing keeps playing way longer after the action ends

**Comment by Agameofscones** [comment]
personal takes:

Most noises that you are using from AvP are from a old stock pack from the early 2000s. I strongly recommend reverting them. They do _not_ sound good. They also remove some iconic noises from the CM audioscape (something something I did this to the smartgun and don't want it to happen again). Also we already have a custom 88m4 firing sound, I don't think it needs replacing.

Explosions are muffled. Ours aren't good persay, but this is not an upgrade.

RE: sound delays, could be filesize.
Make sure all sounds are normalized to **-8db**

Consider getting coder support to make it so items don't play their drop sound when being placed into bags.

New barricade damage sounds are kinda, eeeeh.

**Comment by nauticall** [comment]
> **_Most_** noises that you are using from AvP are from a old stock pack from the early 2000s. I strongly recommend reverting them. They do _not_ sound good. They also remove some iconic noises from the CM audioscape (something something I did this to the smartgun and don't want it to happen again). Also we already have a custom 88m4 firing sound, I don't think it needs replacing.

Most game-ripped sounds were taken from A:CM. If you can state which ones in particular sound bad I'll be happy to have a look, but unless A:CM reused a lot of its sounds from AVP I didn't take anything from AVP at all.

**Comment by nauticall** [comment]
Changelog 7/10:
- Reverted the following sounds: barricade damage, resin breaking, M37 shotgun.
- Added new shotgun pump sounds. Let me know if these are iffy.
- Made it so the "drop" sound only plays when actually dropping something on the floor, not moving it into a container.
- While we're at it, made the defibrillator now *properly* use the empty state when it's empty (i.e. flashing red light).

**Comment by Diegoflores31** [comment]
> most of sounds are a lil to quiet and will make a HUGE difference when you hear a legacy sound 
> xenos breaking barricades its very silent and affects balance

**Comment by stanalbatross** [comment]
a few people have complained about the new pooling sound

**Comment by stanalbatross** [comment]
fix conflict

**Comment by stanalbatross** [comment]
requires sound review from @Agameofscones

**Review by Agameofscones — APPROVED** [review]
nice

**Review by morrowwolf — APPROVED** [review]
CODE LOOKS GOOD, IT IS TIME

**Pull request merged**